suite/tests/client-interface/drreg-test.c

```diff
@@ -44,41 +44,41 @@
 void
 test_asm();
 void
-test_asm_faultA();
+test_asm_fault_restore_gpr();
 void
-test_asm_faultB();
+test_asm_fault_restore_aflags_in_slot();
 void
-test_asm_faultC();
+test_asm_fault_restore_ignore_3rd_dr_tls_slot();
 void
-test_asm_faultD();
+test_asm_fault_restore_non_public_dr_slot();
 void
-test_asm_faultE();
+test_asm_fault_restore_non_public_dr_slot_rip_rel_addr_in_reg();
```

Comment:
I like these names much better than the letters.

Reply:
Yes; made verification easier too.